src/options/options_handler.h

```diff
@@ -128,65 +129,58 @@ class OptionsHandler {
                          const std::string& flag,
                          const std::string& optarg);
 
-  /******************************* main options *******************************/
-  /** Show the solver build configuration and exit */
-  void showConfiguration(const std::string& option, const std::string& flag);
-  /** Show copyright information and exit */
-  void showCopyright(const std::string& option, const std::string& flag);
-  /** Show version information and exit */
-  void showVersion(const std::string& option, const std::string& flag);
-  /** Show all debug tags and exit */
-  void showDebugTags(const std::string& option, const std::string& flag);
-  /** Show all trace tags and exit */
-  void showTraceTags(const std::string& option, const std::string& flag);
+  /******************************* bv options *******************************/
 
-  // theory/bv/options_handlers.h
+  /** Check that abc is enabled */
   void abcEnabledBuild(const std::string& option,
                        const std::string& flag,
                        bool value);
+  /** Check that abc is enabled */
   void abcEnabledBuild(const std::string& option,
                        const std::string& flag,
                        const std::string& value);
-
+  /** Check that abc is enabled */
```

Comment:
Fix this comment?

Reply:
What is wrong with the comment? (the library is actually called abc)